app/views/claims/qts_year.html.erb

```diff
@@ -1,8 +1,22 @@
-<h1 class="govuk-heading-xl">
-  Which academic year were you awarded qualified teacher status (QTS)?
-</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for current_claim, url: claim_path do |form| %>
+      <h1 class="govuk-heading-xl">
```

Comment:
If we wrap the question in a `<fieldset>` and the title in a `<legend>` we make a few accessibility wins. See https://design-system.service.gov.uk/components/fieldset/ for details and the class names.

Reply:
Hmmm, so my reading of that is that the fieldset is to be used when _"you need to show a relationship between multiple form inputs"_. Given that this is a single form input, does it make sense to do that?

Reply:
👍